lib/routes.js

```diff
@@ -1,11 +1,49 @@
-const KoaRouter = require('koa-router')
+const router = require('koa-router')()
 const models = require('./models')
 
-const router = new KoaRouter()
-
 router.get('/', async (ctx) => {
-  await models.dbHeathCheck()
+  await models.dbHealthCheck()
   ctx.body = 'ok'
 })
 
+/* Team */
+
+router
+  .get('/team', async (ctx) => {
```

Comment:
my personal preference is `/teams/`. I think https://developer.github.com/v3/orgs/teams/ this is a really good reference for a good restFul api